docs/src/pages/ui/primitives/passwordfield/react.mdx

```diff
@@ -0,0 +1,345 @@
+import {
+  Flex,
+  PasswordField,
+  TextField,
+  Button,
+  Heading,
+} from '@aws-amplify/ui-react';
+import {
+  PasswordFieldDemo,
+  LoginFormExample,
```

Comment:
Minor, but maybe a pattern we should start doing for the docs: should we split the demo code and examples? It might make the code a bit more organized?

Reply:
That's a great idea, I'll move them to an `examples.ts` file